etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/PostgresDao.java

```diff
@@ -135,13 +143,19 @@ public synchronized PartnerMetadata fetchMetadata(String receivedSubmissionId)
                 timeReceived = timestamp.toInstant();
             }
 
+            String status =
+                    result.getString("delivery_status") != null
+                            ? result.getString("delivery_status")
+                            : "PENDING";
```

Comment:
Oh yeah, I wanted to ask about this.  Is it correct to default to `PENDING`?  Or should we keep it as `null` so that `PartnerMetadata` has `null` for the delivery status?  What was your thinking behind defaulting it to `PENDING`?

Reply:
So I defaulted to `PENDING` for 2 reasons.  1. Not having a status doesn't seem like correct behavior, if we have a message it should have a status. 2. We've set the status column up to be a custom type.  I'm not sure if this would accept a null value when it's a type looking for 1 of 3 specific strings in the db.

Reply:
That makes sense.  Let's not do this defaulting in the db layer but codify it in `PartnerMetadata`, the business class in charge of the enum.  You can add something like this...

```java
public PartnerMetadata {
    if (deliveryStatus == null) {
        deliveryStatus = PartnerMetadataStatus.PENDING;
    }
}
```

Reply:
Okay, sounds good

Reply:
Done